### PR TITLE
e2e test: create account

### PR DESCRIPTION
**Pull request by pivilartisant**
We are adding e2e browser create account  test coverage

### CODE REVIEW

File: web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      compareSnapshot(cy, 'account-create-step-two');
+
+      cy.get('[data-testid="button"]').contains('Define a password').click();
+
+      cy.url().should('eq', `${baseUrl}/account-create-step-three`);
+      compareSnapshot(cy, 'account-create-step-three');
+
+      cy.get('[data-testid="button"]').contains('Skip').click();
+
+      server.createList('account', 1);
```
Comment (mazmassa) [inline web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx:52]: If is only one account you don't need use createList. Just `server.create('account');` should be fine!

File: web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+      cy.url().should('eq', `${baseUrl}/account-create-step-one`);
+
+      cy.get('[data-testid="input-field"]').clear().type('testAccount');
+      cy.get('[data-testid="button"]').contains('Next').click();
+
+      cy.url().should('eq', `${baseUrl}/account-create-step-two`);
+
+      compareSnapshot(cy, 'wrong-account-name-format');
+    });
```
Comment (mazmassa) [inline web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx:87]: I am not sure about this one. 🤔 

We have one test that assert when all goes fine: ✅ 
We have one test that assert when empty field: ✅ 

What would this one add more?

Comment (pivilartisant) [reply web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx:87]: true, I wanted to cover as closely as possible user behavior but if it's not necessary I will delete !

Comment (mazmassa) [reply web-frontend/cypress/e2e/acceptance/account/accountCreate.cy.tsx:87]: >closely as possible user behavior
You did! Both cases. When fails, when not! :)

**Review by mazmassa — APPROVED** [review]

**Review by Thykof — APPROVED** [review]
LGTM

**Pull request merged**